terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1622,6 +1628,36 @@ def qq(*words):
                                     'logfile': '/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log',
                                     'debug': bool(config.debug)
                                 },
+                                'metrics': {
+                                    'metrics_collected': {
+                                        'cpu': {
+                                            'resources': [
+                                                "*"
+                                            ],
+                                            'measurement': [
+                                                'cpu_usage_active'
+                                            ]
+                                        },
+                                        'mem': {
+                                            'measurement': [
+                                                'used_percent'
+                                            ],
+                                        },
+                                        'disk': {
+                                            'measurement': [
+                                                'used_percent'
+                                            ],
+                                            'resources': [
+                                                '/',
+                                                gitlab_mount
+                                            ],
+                                            'drop_device': True
```

Comment:
What does this do?

Reply:
>drop_device – Setting this to true causes Device to not be included as a dimension for disk metrics.
Preventing Device from being used as a dimension can be useful on instances that use the Nitro system because on those instances the device names change for each disk mount when the instance is rebooted. This can cause inconsistent data in your metrics and cause alarms based on these metrics to go to INSUFFICIENT DATA state.
The default is false.

TBH, I've left this as it was provided in the example. However, if I understood correctly, not dropping it would include an additional dimension that (I think) is redundant since is just a constant variable.